project_generation/content/templates/base-app/Makefile.tmpl

```diff
@@ -11,7 +11,7 @@ all: audit test build
 
 .PHONY: audit
 audit:
-	nancy go.sum
+	go list -m all | nancy sleuth
```

Comment:
Should we be doing `go list -json -m all` here. Ref: https://github.com/sonatype-nexus-community/nancy.

Also should we consider introducing the facility to have few exclusion of vulnerabilities?
for eg:
`go list -json -m all | nancy sleuth --exclude-vulnerability-file ./.nancy-ignore`

Reply:
makes sense thanks